app/controllers/provider_interface/provider_interface_controller.rb

```diff
@@ -33,7 +33,10 @@ def current_provider_user
     end
 
     def check_cookie_preferences
-      @google_analytics_id = ENV.fetch('GOOGLE_ANALYTICS_MANAGE', '') if cookies['consented-to-manage-cookies'].eql?('yes')
+      if cookies['consented-to-manage-cookies'].eql?('yes')
+        @google_analytics_id = ENV.fetch('GOOGLE_ANALYTICS_MANAGE', '')
+        @google_tag_manager_id = ENV.fetch('GOOGLE_TAG_MANAGER_MANAGE', '')
```

Comment:
do we want to add this to the .env files as well, similar to `GOOGLE_ANALYTICS_MANAGE`?

Reply:
We seem to be surfacing the analytics id on the cookies page: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/views/content/cookies.html.erb#L114
I wonder if we should do the same for the tag manager (not sure) 🤔